resources/public/css/style.css

```
@@ -1,2 +1,95 @@
 /* some style */
 
+/* body { */
+/*     width: 100%; */
+/* } */
+
+/* fonts */
+html, button, input, select, textarea,
+.pure-g [class *= "pure-u"] {
+    font-family: "Helvetica Neue", sans-serif;
+}
+
+.content {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 768px;
+}
+
+.manual {
+    border: solid;
+    padding: 2em;
+}
+
+.button-delete {
+    color: white;
+    /* text-shadow: 0 1px 1px rgba(0, 0, 0, 0.2); */
+    background: rgb(202,60,60);
+}
+
+textarea {
+    max-width: 100%;
+}
+
+.controls {
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 1em;
+    width: max-content;
+}
+
+.forest {
+    display: flex;
+    flex-direction: row;
+    width: min-content;
+}
+
+.tree {
+    margin-left: auto;
+    margin-right: auto;
+    /* position: relative; */
+}
+
+.subtree {
+    /* margin-top: 1px; */
+    margin-right: 2px;
+    flex-grow: 1;
+    flex-basis: auto;
+}
+
+.subtree:last-child {
+    margin-right: 0;
+}
+
+.children {
+    margin-bottom: 2px;
+    width: 100%;
+}
+
+.node {
+    /* position: absolute; */
+    border-style: solid;
+    border-width: thin;
+    border-color: #555;
+    text-align: center;
+    min-width: 60px;
```

```suggestion
    min-width: 10px;
```

10px are totally enough here. For long sequences like in real world examples, it is important to save as much horizontal space as possible.